packages/substrate-client/src/chainhead/chainhead.ts

```diff
@@ -120,13 +116,35 @@ export function follow(
               }
             : undefined,
         )
-      })
+      }
+
+      if (typeof followSubscription === "string")
+        followSubscriptionCb(followSubscription)
+      else followSubscription.then(followSubscriptionCb)
 
       return () => {
         onCancel()
       }
     }
 
+    const storage = createStorageFn(fRequest)
+
+    // System::BlockHash[0] using Twox64Concat
+    const SystemBlocHashAtZero =
```

Comment:
```suggestion
    const SystemBlockHashAtZero =
```

Reply:
I ended up removing it, though... But thanks!